mappings/net/minecraft/block/Material.mapping

```diff
@@ -1,26 +1,64 @@
-	FIELD field_26185 CLAY Lnet/minecraft/class_1241;
+	FIELD field_26184 CACTUS Lnet/minecraft/class_1241;
+	FIELD field_26185 ORGANIC_PRODUCT Lnet/minecraft/class_1241;
+	FIELD field_26186 GOURD Lnet/minecraft/class_1241;
+	FIELD field_26187 EGG Lnet/minecraft/class_1241;
+	FIELD field_26188 PORTAL Lnet/minecraft/class_1241;
+	FIELD field_26189 CAKE Lnet/minecraft/class_1241;
+	FIELD field_26190 COBWEB Lnet/minecraft/class_1241;
+	FIELD field_26191 PISTON Lnet/minecraft/class_1241;
+	FIELD field_26192 BARRIER Lnet/minecraft/class_1241;
+	FIELD field_26193 STRUCTURE_VOID Lnet/minecraft/class_1241;
+	FIELD field_26194 burnable Z
+	FIELD field_26195 replaceable Z
+	FIELD field_26196 allowsLight Z
 	FIELD field_26197 color Lnet/minecraft/class_1243;
-	FIELD field_26199 PISTON Lnet/minecraft/class_3686;
+	FIELD field_26198 breakByHand Z
+	FIELD field_26199 pistonBehavior Lnet/minecraft/class_3686;
+	FIELD field_26200 ignoreAdventureMode Z
 	FIELD field_26201 AIR Lnet/minecraft/class_1241;
-	FIELD field_26202 GRASS Lnet/minecraft/class_1241;
-	FIELD field_26203 DIRT Lnet/minecraft/class_1241;
+	FIELD field_26202 SOLID_ORGANIC Lnet/minecraft/class_1241;
+	FIELD field_26203 SOIL Lnet/minecraft/class_1241;
 	FIELD field_26204 WOOD Lnet/minecraft/class_1241;
 	FIELD field_26205 STONE Lnet/minecraft/class_1241;
-	FIELD field_26206 IRON Lnet/minecraft/class_1241;
-	FIELD field_26207 METAL Lnet/minecraft/class_1241;
+	FIELD field_26206 METAL Lnet/minecraft/class_1241;
+	FIELD field_26207 REPAIR_STATION Lnet/minecraft/class_1241;
 	FIELD field_26208 WATER Lnet/minecraft/class_1241;
 	FIELD field_26209 LAVA Lnet/minecraft/class_1241;
-	FIELD field_26210 FOILAGE Lnet/minecraft/class_1241;
-	FIELD field_26213 YELLOW Lnet/minecraft/class_1241;
-	FIELD field_26214 WEB Lnet/minecraft/class_1241;
+	FIELD field_26210 LEAVES Lnet/minecraft/class_1241;
+	FIELD field_26211 PLANT Lnet/minecraft/class_1241;
+	FIELD field_26212 REPLACEABLE_PLANT Lnet/minecraft/class_1241;
+	FIELD field_26213 SPONGE Lnet/minecraft/class_1241;
+	FIELD field_26214 WOOL Lnet/minecraft/class_1241;
+	FIELD field_26215 FIRE Lnet/minecraft/class_1241;
 	FIELD field_26216 AGGREGATE Lnet/minecraft/class_1241;
 		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
-	FIELD field_26224 ICE Lnet/minecraft/class_1241;
-	FIELD field_26225 WHITE Lnet/minecraft/class_1241;
+	FIELD field_26217 SUPPORTED Lnet/minecraft/class_1241;
+	FIELD field_26218 CARPET Lnet/minecraft/class_1241;
+	FIELD field_26219 GLASS Lnet/minecraft/class_1241;
+	FIELD field_26220 REDSTONE_LAMP Lnet/minecraft/class_1241;
+	FIELD field_26221 TNT Lnet/minecraft/class_1241;
+	FIELD field_26222 UNUSED_PLANT Lnet/minecraft/class_1241;
+	FIELD field_26223 ICE Lnet/minecraft/class_1241;
+	FIELD field_26224 DENSE_ICE Lnet/minecraft/class_1241;
+	FIELD field_26225 SNOW_LAYER Lnet/minecraft/class_1241;
+	FIELD field_26226 SNOW_BLOCK Lnet/minecraft/class_1241;
```

Comment:
`SNOW`

Reply:
he named it correctly

Reply:
But its called SNOW_LAYER and SNOW_BLOCK in 1.16 too

Reply:
no point in replying to a resolved conversation 😛